src/effects/VST/VSTEffect.cpp

```diff
@@ -1477,6 +1477,8 @@ void VSTEffect::ExportPresets(const EffectSettings &) const
       return;
    }
 
+   StoreSettings(GetSettings(settings));
```

Comment:
In the last commit, which adds two calls to `StoreSettings`, check the return value and return early if it is false.  In `SaveParameters`, do that before any other steps changing the config file.

Reply:
That's right, done.